interfaces/IZigWave.h

```diff
@@ -23,11 +23,19 @@
 
 namespace WPEFramework {
 	namespace Exchange {
+
+        /* @json */
         struct EXTERNAL IZigWave : virtual public Core::IUnknown {
             enum { ID = ID_ZIGWAVE };
 
-            virtual void Include(const string& config) = 0;
-            virtual void Exclude(const string& config) = 0;
+            // @property
+            // @brief To allow new devices to the network, the controller should be placed
+            //        into an accepting mode. By enabling this mode, the controller can 
+            //        accept new devices.
+            // @param enabled (true) or disable (false) the accepting mode.
+            // 
+            virtual uint32_t Accepting(bool& enabled /* @out */) const = 0;
```

Comment:
Mhmm, did you consider using the interface like this might not lead to what the "user" might expect:

bool b;
IZigWave* p = bla...;
p->Accepting(b);

Of course also a bit a problem of incoprrect usage but perhaps better to guard for that by choosing a different name?

Reply:
After discussing understand why we do this